workers/unity/Assets/Gdk/Core/Worker/WorkerBase.cs

```diff
@@ -19,18 +19,22 @@ public abstract class WorkerBase : IDisposable
 
         public abstract string GetWorkerType { get; }
 
-        protected WorkerBase(string workerId, Vector3 origin)
+        protected WorkerBase(string workerId, Vector3 origin) : this(workerId, origin, new LoggingDispatcher())
```

Comment:
remove this constructor and `IErrorDispatcher errorDispatcher` optional / assign it to `new LoggingDispatcher()` as the default value